UI/Shared/PlayerCard.razor

```diff
@@ -0,0 +1,25 @@
+<div class="bg-indigo-900 h-screen grid content-between justify-items-center p-2"> 
+    <div class="rounded-full overflow-hidden h-80 w-80">
+        <img src="@Image" />
+    </div>
+    <div class="text-3xl font-semibold w-31 h-19">
+        <h1 class="text-white">@Name</h1>
```

Comment:
Vertical spacing between the photo, the name and the mark needs to be improved. See Figma.